src/Control/Biegunka/Source/Git/Internal.hs

```diff
@@ -127,6 +127,24 @@ updateGit u p Git { _branch, _failIfAhead } =
              return (Just (printf "‘none’ → ‘%s’" after))
         )
 
+checkBranch :: Maybe String -> String -> IO ()
+checkBranch maybeCurrentBranch remoteBranch =
```

Comment:
If you flip the order of the arguments you can avoid having this awkward variable name with a `\case`.